### PR TITLE
Feature/GitHub actions

### PR DESCRIPTION
**Pull request by GregVaz**
Configuration for github actions

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Rails tests CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
```
Comment (OmauryMarmolejo) [inline .github/workflows/main.yml:8]: What happens if I can push some new code to my current branch and that branch is not master?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```
Comment (OmauryMarmolejo) [inline .github/workflows/main.yml]: Is this line needed?

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      - name: Install dependencies
+        run: 
+          bundler install
+      - name: Create database
+        run: |
+          bundler exec rails db:create RAILS_ENV=test
+          bundler exec rails db:migrate RAILS_ENV=test
+      - name: Run tests
+        env:
+          OPEN_WEATHER_KEY: 866b5463b979e3ffb0a4f566137f898f
```
Comment (OmauryMarmolejo) [inline .github/workflows/main.yml]: Please try to keep safe the keys...

Comment (GregVaz) [reply .github/workflows/main.yml]: Sure, I'll implement Github Environment Secrets

File: .github/workflows/main.yml (hunk 2 above)
Comment (GregVaz) [reply .github/workflows/main.yml]: No, it is not needed, thanks

File: .github/workflows/main.yml (hunk 1 above)
Comment (GregVaz) [reply .github/workflows/main.yml:8]: Ok, thanks for the observation. I will implement the push event only for branches with name 'feature/**'

**Pull request merged**